Procurement/ViewModel/Recipes/Recipe.cs

```diff
@@ -38,5 +39,13 @@ public virtual IEnumerable<RecipeResult> Matches(IDictionary<Tab, List<Item>> it
             }
             return Matches(flatItems);
         }
+
+        protected virtual bool GetShouldUseShortRecipeDescriptions()
```

Comment:
Doesn't need to be virtual - it's not overridden.

Reply:
Done.  I usually err on the side of making more stuff virtual so it's easier down the road, but, indeed, it isn't being overridden right now.